### DISCUSSION TITLE
Provide a way of saying "Thank you" via emoji reaction

### DISCUSSION THREAD

**Discussion by rahim** [community/community#38201] [Pull Requests]
In one sense this could be considered covered by the suggestion to allow all emoji as reactions in https://github.com/community/community/discussions/10240

"Thank you" is such an important sentiment to express that the current reaction palette lacks that it felt worth a separate request.

There's also another subtle issue here, there's not perfect consensus across the industry of how thanks should be represented. Most often this is covered by [Folded Hands](https://emojipedia.org/folded-hands/) 🙏 (which is only keyed as `:pray:` in Github, there is no `:thanks:` today).

**Comment by ossanna16** [comment]
Hi there @rahim and welcome to our community! Thank you for the feedback 🙂

**Comment by el-qq** [comment]
+1. Very lacking for answers

**Comment by mdtanvirahamedshanto** [comment]
welcome to our community! Thank you for the feedback

**Comment by leovct** [comment]
+1

**Comment by bryevdv** [comment]
+1 to this. Wanting to thank my colleagues or a new contributor is a near daily occurrence. Wanting to send them a little heart *is not*. 

🙏 would be a great addition for this purpose.

**Comment by xiankaing** [comment]
+1 please add :pray:
